CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,76 @@
+# Contributor Covenant Code of Conduct
```

Comment:
```suggestion
# Reviewdog Code of Conduct
```
or maybe
```suggestion
# Reviewdog community Code of Conduct
```
I looked at [their examples](https://www.contributor-covenant.org/adopters/) and it seems like a lot of people leave `Contributor Covenant` in, but I don't think their meant to...? Seems weird... Quite a few people replace it too.
There's already an `Attribution` section at the bottom, so I think it's obvious where it comes from.